### PR TITLE
Added a section in the Control Panel article regarding reusing the Co…

### PR DESCRIPTION
**Pull request by StefanNenchev**
…ntrol Panel for the different RadGridView controls within the app

### CODE REVIEW

**Comment by PetyaSotirova** [comment]
Enumerate the code snippets and add images demosntrating how the control panel looks. Also, add a _See Also_ section at the bottom with links to relevant topics.

File: controls/radgridview/features/overview-controlpanel.md [hunk 1]
```diff
@@ -65,4 +65,98 @@ For example:
 
 
 >tipYou can download a __runnable solution__ showing how to define ControlPanelItems from our online SDK repository [here](https://github.com/telerik/xaml-sdk/), the example is listed under __GridView/ControlPanel__.
```
Comment (PetyaSotirova) [inline controls/radgridview/features/overview-controlpanel.md]: Update with direct link to the example.

File: controls/radgridview/features/overview-controlpanel.md [hunk 2]
```diff
@@ -65,4 +65,98 @@ For example:
...
-    
+   
+
+## Reusing Control Panel Items  ##
+
+As of __Q1 2016__ we have added a DataTemplate support for the __ButtonContent__ and __Content__ properties of the __ControlPanelItem__. Developers have the ability to add a common Style targeting the ControlPanel that all the RadGridView controls in the application will inherit.
```
Comment (PetyaSotirova) [inline controls/radgridview/features/overview-controlpanel.md]: As of Q1 2016 RadGrdiView supports DataTemplates for the ButtonContent and Content properties of the ControlPanelItem. You have the ability...

**Comment by erikruth** [comment]
Can you give me a link to the articles you want me to edit? I'm only seeing an option to Merge pull request.

**Pull request merged**